spec.yaml

```diff
@@ -89,10 +89,16 @@ paths:
                 "code": "RESOURCE_CREATED",
                 "message": "A new cluster has been created with ID 'wqtlq'"
               }
-        default:
-          description: Error
+        "401":
+          description: Permission Denied
```

Comment:
Without "default" we would have to describe every possible response (status code) in every operation, to create a valid spec that could be used to create functioning clients.

If you use the spec to create a client, and the API operation returns a status code not contained in the spec (e.g. 500), and there is no "default" response for the operation, the client can't handle it.

With "default" we can just describe that there are responses with status codes other then the ones described explicitly here, all using the same response body format (V4GenericResponse). As `description` we can probably use something like `Other error` or maybe we can even omit `description`.